src/sentry/api/serializers/models/filechange.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import absolute_import
+
+import six
+
+from sentry.api.serializers import Serializer, register
+from sentry.models import CommitFileChange
+from sentry.api.serializers.models.release import get_users_for_commits
+
+
+@register(CommitFileChange)
+class CommitFileChangeSerializer(Serializer):
+    def get_attrs(self, item_list, user):
+        commits = [f.commit for f in item_list]
+        author_objs = get_users_for_commits(commits)
+        result = {}
+        for item in item_list:
+            result[item] = {
+                'user': author_objs.get(item.commit.author_id, {})
```

Comment:
This will need to be changed up to be a bit different too to actually use the `commits` you just queried instead. Probably with a mapping of Commit objects by id.